apps/prairielearn/src/middlewares/clearCookies.js

```diff
@@ -1,19 +1,27 @@
 const _ = require('lodash');
 
+const { clearCookie } = require('../lib/cookie');
+
 const cookies_to_ignore = [
   'pl_authn',
+  'pl2_authn',
   'pl_assessmentpw',
+  'pl2_assessmentpw',
   'pl_access_as_administrator',
+  'pl2_access_as_administrator',
   'pl_disable_auto_authn',
+  'pl2_disable_auto_authn',
+  'prairielearn_session',
```

Comment:
I don't think we need this here because it doesn't start with `pl_`/`pl2_` so we will already skip it. It's fine to leave it here for consistency, however.

Reply:
Good point! I will leave it for consistency though.